hybris/common/q/dlfcn.cpp

```diff
@@ -300,8 +300,13 @@ void __loader_remove_thread_local_dtor(void* dso_handle) {
   decrement_dso_handle_reference_counter(dso_handle);
 }
 
+__LIBC_HIDDEN__ libc_shared_globals* __libc_shared_globals() {
+  static libc_shared_globals globals;
+  return &globals;
+}
+
 libc_shared_globals* __loader_shared_globals() {
-  return __libc_shared_globals();
+   return __libc_shared_globals();
```

Comment:
Nitpick: useless indentation change.

Reply:
Done, and found many other similar style problems, fixed too.